examples/dmm/dmm.py

```diff
@@ -156,7 +156,7 @@ def __init__(self, input_dim=88, z_dim=100, emission_dim=100,
                           dropout=rnn_dropout_rate)
 
         # if we're using normalizing flows, instantiate those too
-        self.iafs = [AffineAutoregressive(AutoRegressiveNN(z_dim, [iaf_dim])) for _ in range(num_iafs)]
+        self.iafs = [affine_autoregressive(z_dim, hidden_dims=[iaf_dim]) for _ in range(num_iafs)]
```

Comment:
this change should probably be mirrored in dmm.ipynb

Reply:
Could we leave this for a separate PR after the 1.0 release?